napari/_qt/qt_preferences.py

```diff
@@ -0,0 +1,60 @@
+"""
+"""
+
+from qtpy.QtCore import Signal
+from qtpy.QtWidgets import (
+    QDialog,
+    QHBoxLayout,
+    QListWidget,
+    QPushButton,
+    QStackedWidget,
+    QVBoxLayout,
+)
+
+
+class PreferencesDialog(QDialog):
+    sig_resized = Signal(object)
+
+    def __init__(self, parent):
+        super().__init__(parent)
+
+        self._list = QListWidget(self)
+        self._stack = QStackedWidget(self)
+
+        # Use a Buttons group (forgot the name)
+        self._button_cancel = QPushButton("Ok")
+        self._button_ok = QPushButton("Cancel")
```

Comment:
labels are backwards

Reply:
This was a stub for demo purposes. But indeed they are backwards!